_opcodes/op350-pstee.html

```diff
@@ -0,0 +1,16 @@
+---
+n: 350
+opname: "Guardian Mantle (Non-cumulative)"
+param1: "Enabled?"
+param2: "Unused"
+bg1: 0
+bg2: 0
+bgee: 0
+iwd1: 0
+iwd2: 0
+pst: 0
+pstee: 1
+---
+This effect protects the target creature from physical damage.
```

Comment:
In the original it required the attacker to succeed on a save vs spells (at -4). On failure no damage was dealt. But it's true that it's handled elsewhere, not in the opcode itself.

Reply:
Possibly. But I can't verify it, as this and several other opcodes appear to be used internally by the respective spells (probably via [opcode 375](https://gibberlings3.github.io/iesdp/opcodes/pstee.htm#op375)).

Reply:
This is from Avenger's reverse engineering; I doubt this got changed for PST:EE.

Reply:
Yes, the saving throw to bypass guardian mantle is hardcoded into op12.

It's been a while since I looked at the op375-esc opcodes, but iirc they aren't used internally. They are duplicates of op375 functionality that have been extracted into dedicated opcodes. I've attached my full notes here if it helps. Looks like the dedicated opcodes attach their visual effects differently; instead of using an internal PST-style projectile like op375, they attach a VVC?

[Research - PSTEE.txt](https://github.com/Gibberlings3/iesdp/files/11023280/Research.-.PSTEE.txt)

Reply:
I can confirm that the saving throw part is hardcoded.

> instead of using an internal PST-style projectile like op375, they attach a VVC?

BAM resrefs work just as well.